OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -145,12 +146,16 @@ public void setSocialLoginButtons(boolean googleEnabled, boolean facebookEnabled
                                               boolean microsoftEnabled) {
                 if (!facebookEnabled && !googleEnabled && !microsoftEnabled) {
                     activityLoginBinding.panelLoginSocial.setVisibility(View.GONE);
-                } else if (!facebookEnabled) {
-                    activityLoginBinding.socialAuth.facebookButton.getRoot().setVisibility(View.GONE);
-                } else if (!googleEnabled) {
-                    activityLoginBinding.socialAuth.googleButton.getRoot().setVisibility(View.GONE);
-                } else if (!microsoftEnabled) {
-                    activityLoginBinding.socialAuth.microsoftButton.getRoot().setVisibility(View.GONE);
+                } else {
```

Comment:
why was this logic changed? previous logic seems fine to me

Reply:
if Facebook and google is disabled, but this logic only hides facebook, not the google button.